book/internals/setters.nim

```diff
@@ -0,0 +1,76 @@
+import nimib, nimibook
+
+nbInit(theme = useNimibook)
+
+nbText: """
+## **Setters**
+Setters are a convenience mechanism for widgets.
+Through them you can hide fields on your `WidgetState`.
```

Comment:
This might be misleading. The fields can still be accessed.

Reply:
Agreed, I removed that mention and rewrote the later section that also talks about "hiding" stuff.
Speaking of "Hiding" in that sense really only makes sense if we either want to explicitly discourage assigning directly to fields with setters, or implement a mechanism that directly blocks it.